src/app/tasks/task-list/task-list.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input } from '@angular/core';
+import { Task } from '../task.model';
+import { TaskItemComponent } from '../task-item/task-item.component';
+import { TaskList } from '../task-list.model';
+
+@Component({
+  selector: 'app-task-list',
+  standalone: true,
+  imports: [
+    TaskItemComponent
+  ],
+  templateUrl: './task-list.component.html',
+  styleUrl: './task-list.component.scss'
+})
+export class TaskListComponent {
+
+  @Input() taskList!: TaskList|undefined;
```

Comment:
Don’t use non null assertion (!) you can define the input like this: 
taskList: TaskList | null = null